src/battle.ts

```diff
@@ -2778,6 +2802,30 @@ class Battle {
 			this.log(args, kwArgs);
 			break;
 		}
+		case '-sideswitch': {
+			let effect = Dex.getEffect(args[1]);
```

Comment:
Are we sending this individually for each condition? It might make sense to switch them all at once?